.github/workflows/meta-tuist.yml

```
@@ -73,7 +73,7 @@ jobs:
           bundle install --jobs 4 --retry 3
       - name: Build
         run: |
-          ./fourier focus
+          ./fourier focus --source
```

I would rename it (and all the others) to `--from-sources`
```suggestion
          ./fourier focus --from-sources
```